brave/brave-core
# Implement Brave Ads inline content ads

Resolves https://github.com/brave/brave-browser/issues/16008

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Requires regression test across ad notification delivery
- Confirm frequency caps are working as expected for new tab page ads (Sponsored Images)
- Confirm users are rewarded for viewing new tab page ads (Sponsored Images)
- Confirm frequency caps are working as expected for promoted content ads (Brave News Ads)
- Confirm users are rewarded for viewing promoted content ads (Brave News Ads)
- Confirm frequency caps are working as expected for inline content ads (Brave News Ads)
- Confirm ad pacing is working as expected for inline content ads (Brave News Ads)
- Confirm ad priority is working as expected for inline content ads (Brave News Ads)
- Confirm users are rewarded for viewing inline content ads (Brave News Ads)
